gh-138775: fix handle `python -m base64` stdin correct with EOF signal (#138776)
python/cpython: Misc/NEWS.d/next/Library/2025-09-11-15-03-37.gh-issue-138775.w7rnSx.rst

> Fix: handle "python -m base64" stdin correct with EOF single.

Reviewer: ```suggestion
:mod:`base64`: fix reading from stdin for the command-line interface.
```

Author: learned that will try to learn the synax here

> :mod:`base64`: fix reading from stdin for the command-line interface.

Reviewer 2: "fix reading" is too vague. We should say what the bug was, not how it was fixed. How about this?
```suggestion
Fix the EOF signal being ignored in the :mod:`base64` command-line interface.
```

Reviewer: I would suggest keeping `mod base64` at the beginning. It helps categorizing the changelog entry for readers. Also, EOF is not a signal.

Reviewer 2: Yeah, I don't have a better term for it. It works with EOF, but not CTRL^D.

Author: so keep old one is OK?

Reviewer 2: No, it's too vague.

Author: refer it, with the suggestion
